src/main/kotlin/model/HMPPSDomainEvents.kt

```diff
@@ -0,0 +1,57 @@
+package uk.gov.justice.hmpps.architecture
+
+import com.structurizr.model.*
```

Comment:
I think it's generally explicit imports instead of wildcards around here so I think that's just:
```
import com.structurizr.model.Model
import com.structurizr.model.SoftwareSystem
```

Reply:
This reminded me to add ktlint, which is now in #109